apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -2,17 +2,17 @@ import { QuizQuestion } from "~/libs/components/components.js";
 import { useAppForm } from "~/libs/hooks/hooks.js";
 
 type FormValues = {
```

Comment:
Looks similar to a default payload for signIn or signUp form. For these components, default payload was moved into libs/constants/constants.ts file for each component respectively

Reply:
I don't use this component in my ticket.